torch_xla/csrc/ops/ops_lower_fn.cpp

```diff
@@ -112,6 +112,31 @@ torch_xla::XlaOpVector Ceil::Lower(LoweringContext* loctx) const {
   return ReturnOp(xla::Ceil(xla_input), loctx);
 }
 
+torch_xla::XlaOpVector ClampTensor::Lower(LoweringContext* loctx) const {
+  XLA_CHECK(has_min || has_max)
+      << "At least one of \'min\' or \'max\' must not be None";
+
+  xla::XlaOp res = loctx->GetOutputOp(operand(0));
+  if (has_min && has_max) {
+    auto promoted_min =
+        XlaHelpers::Promote(res, loctx->GetOutputOp(operand(1)));
+    res = xla::Max(promoted_min.first, promoted_min.second);
+    auto promoted_max =
+        XlaHelpers::Promote(res, loctx->GetOutputOp(operand(2)));
+    res = xla::Min(promoted_max.first, promoted_max.second);
+  } else if (has_min) {
+    auto promoted_min =
+        XlaHelpers::Promote(res, loctx->GetOutputOp(operand(1)));
+    res = xla::Max(promoted_min.first, promoted_min.second);
+  } else if (has_max) {
+    auto promoted_max =
+        XlaHelpers::Promote(res, loctx->GetOutputOp(operand(1)));
+    res = xla::Min(promoted_max.first, promoted_max.second);
+  }
```

Comment:
This is little bit ugly due to `min` and `max` tensors being optional (`operand[1]` can be either `min` or `max`):
- if !has_min and has_max, operand[1] is `max`
- if has_min and !has_max, operand[1] is `min`